components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -23,6 +25,96 @@
+For details, see: https://docs.yscope.com/clp/main/user-guide/telemetry
+
+You can disable metrics at any time by setting CLP_DISABLE_TELEMETRY=true
+or by blocking https://telemetry.yscope.io at the network level.
+
+Enable anonymous telemetry to help improve CLP? [Y/n]
+================================================================================
+"""
+
+
+def _check_telemetry_consent(
+    clp_config, config_file_path: pathlib.Path, clp_home: pathlib.Path
```

Ruff (0.15.12)

[warning] Unused function argument: `clp_home`

(ARG001)